src/main/java/ch/njol/skript/events/EvtJump

```diff
@@ -0,0 +1,39 @@
+package ch.njol.skript.events;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.lang.Literal;
+import ch.njol.skript.lang.SkriptEvent;
+import ch.njol.skript.lang.SkriptParser;
+import org.bukkit.Statistic;
+import org.bukkit.event.Event;
+import org.bukkit.event.player.PlayerStatisticIncrementEvent;
+import org.eclipse.jdt.annotation.Nullable;
+
+public class EvtJump extends SkriptEvent {
+
+    static {
+        Skript.registerEvent("Jump", EvtJump.class, PlayerStatisticIncrementEvent.class, "[player] jump[ing]")
+					.description("Called whenever a player jumps.")
+					.examples("on jump:",
+							"	event-player does not have permission \"jump\"",
+							"	cancel event")
+					.since("2.3 (Paper), 2.? (Spigot)")
```

Comment:
For the 2.?, use INSERT VERSION, it all gets changed before a release is made

Reply:
kk